3.10/arangosync-migration-tool.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+description: >-
+  Migrating data from bare metal servers to the cloud with minimal downtime
+---
+# Cloud Migration Tool
+
+{{ page.description }}
+{:class="lead"}
+
+The `arangosync-migration` tool allows you to easily migrate data to the
+cloud while ensuring a smooth transition with minimal downtime.
+Start the cloud migration, let the tool do the job and, at the same time,
+keep your local cluster up and running. 
+
+Some of the key benefits of the cloud migration tool include:
+- Safety comes first - pre-checks and potential failures are carefully handled.
+- Your data is secure and fully encrypted.
+- Ease-of-use with a live migration while your local cluster is still in use.
+- Get access to what a cloud-based fully managed service has to offer: 
+high availability and reliability, elastic scalability, and much more.
+
+## Prerequisites 
+
+Before getting started, make sure the following prerequisites are in place:
+
+- Go to [Oasis Cloud](https://cloud.arangodb.com/home){:target="_blank"}
+and sign in. If you don’t have an account yet, sign-up to create one.
+
+- Generate an Oasis API key and API secret. See a detailed guide on 
+[how to create an API key](oasis/api-getting-started.html#creating-an-api-key).
+
+### Setting up the target deployment in Oasis
+
+Continue by [creating a new Oasis deployment](oasis/deployments.html#how-to-create-a-new-deployment)
+or choose an existing one.
+
+The target deployment in Oasis requires specific configuration rules to be
+set up before the migration can start:
+
+- **Configuration settings**: The target deployment must be compatible with the
+source data cluster. This includes the ArangoDB version that is being used,
+the DB-Servers count, and disk space.
+- **Deployment region and cloud provider**: Choose the closest region to your
+data cluster. This factor can speed up your migration to the cloud.
+
+After setting up your Oasis deployment, wait for a few minutes to become 
+fully operational.
+
+## Running the migration tool
+
+The `arangosync-migration` tool provides a set of commands that allow you to:
+-  start the migration process
+-  check whether your source and target clusters are fully compatible
+-  get the current status of the migration process
+-  stop or abort the migration process
+-  switch the local cluster to read-only mode
+
+### Starting the migration process
+
+To start the migration process, run the following command:
+
+```bash
+arangosync-migration start
+```
+
+An optional `--check-only` option can be provided. When specified, you can check
+if your local cluster and target deployment are compatible and if the migration 
+can start. No data is sent to Oasis.
+
+Before the actual migration starts, the `start` command also runs some
```

Comment:
This sentence is kind of copying the same info from previous paragraph.

I think it needs to be reworked a bit:
checks are always executed (regardless of provided flags), but customer can provide '--check-only' option so migration will not start if checks are passed.